src/components/schedule-page/table/schedule/schedule.component.tsx

```diff
@@ -17,37 +19,119 @@ export function ScheduleComponent(): JSX.Element {
     setNewSchedule(scheduleModel);
   }, [scheduleModel, setNewSchedule]);
 
+  function isPresent(): boolean {
+    if (scheduleLocalState.isInitialized) {
+      const a = scheduleLocalState.dateSection?.length;
+      const b = scheduleLocalState.nurseShiftsSection?.length;
+      const c = scheduleLocalState.babysitterShiftsSection?.length;
+
+      if (a && b && b && c && a * b * c) {
+        return true;
+      }
+    }
+    return false;
+  }
+
   return (
     <React.Fragment>
-      {scheduleLocalState.isInitialized && (
-        <table className="table">
+      {!isPresent() && (
+        <table>
+          <tbody>
+            <tr>
+              <td></td>
```

Comment:
Czy nie da się tego zrobić jakoś inaczej niż za pomocą pustych <td></td>. Bo gdy je usuwam to dostaję coś takiego:
![image](https://user-images.githubusercontent.com/23723122/100877750-81feaa00-34a9-11eb-99d4-7b73afd082ce.png)